### PR TITLE
Fix ci tests

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cid/helpers/quicksight/__init__.py [hunk 1]
```diff
@@ -89,7 +89,10 @@ def identityRegion(self) -> str:
                     raise
             except Exception as e:
                 logger.info(f'QuickSight identity region detection failed, using {self.region}')
-                logger.debug(e, stack_info=True)
+                if 'ResourceNotFoundException' in str(e):
+                    logger.debug(f'ResourceNotFound {self.username}, but it can be ok')
+                else:
+                    logger.debug(e, stack_info=True)
```
Comment (iakov-aws) [inline cid/helpers/quicksight/__init__.py:95]: TODO: manage ResourceNotFoundException

Comment (iakov-aws) [reply cid/helpers/quicksight/__init__.py]: @darken99 this is done

**Comment by darken99** [comment]
Let's move datasource selection logic and CI/CD logic change into separate PRs

**Comment by iakov-aws** [comment]
> Let's move datasource selection logic and CI/CD logic change into separate PRs

Datasource selection is one of the fails that this PR is fixes. without that cases will be still broken. Please approve and feel free to propose other refactoring on top of that

**Comment by iakov-aws** [comment]
@darken99  here is the datasource PR  https://github.com/aws-samples/aws-cudos-framework-deployment/pull/386

**Review by darken99 — APPROVED** [review]

**Comment by iakov-aws** [comment]
@darken99  Thx

**Pull request merged**